remove-testing-images/package.json

```diff
@@ -9,7 +9,8 @@
     },
     "dependencies": {
         "@actions/core": "1.10.0",
-        "@actions/github": "5.1.1"
+        "@actions/github": "5.1.1",
+        "moment": "^2.29.4"
```

Comment:
This works alright but I'd prefer to pick something different than moment because of:

<img width="856" alt="Screenshot 2023-10-31 at 14 53 14" src="https://github.com/wunderflats/actions/assets/6202537/3f24c6a4-53c3-4bb2-87b8-bbff6c08469c">

An alternative that we use in other parts of the code is `date-fns` (fairly straightforward to learn). In another place we use `@js-temporal/polyfill` (takes a bit more thinking to get used to the concepts).